src/GameState/index.tsx

```diff
@@ -112,9 +111,21 @@ export default function GameState({ navigation, route, player }: any) {
     }
   };
 
-  const getPlayerObject = (player: User) => {
+  const getMobileUserInformation = () => {
```

Comment:
You could have one method, passing the user id as param. Also, I found the function name confusing, are we getting information specific for mobile? I would go for a `getUserInformation(userId)`.